packages/core/package.json

```diff
@@ -101,12 +101,14 @@
     "isomorphic-fetch": "^3.0.0",
     "karma": "^6.4.2",
     "karma-chrome-launcher": "^3.2.0",
+    "karma-firefox-launcher": "^2.1.2",
```

Comment:
Should we add these packages in the monorepo root file as all/most packages will use them?

Reply:
I tried it initially and got some conflicts. But in general yes, we should keep them at the root level to avoid discrepancies in versions.